pkg/test/framework/components/istio/config.go

```diff
@@ -56,6 +56,26 @@ const (
 	DefaultCIUndeployTimeout = time.Second * 900
 )
 
+// ControlPlaneTopology defines a supported control plane topologies.
+type ControlPlaneTopology string
+
+const (
+	// ReplicatedControlPlane is a ControlPlaneTopology where each cluster has its own control plane.
+	ReplicatedControlPlane ControlPlaneTopology = "ReplicatedControlPlane"
+
+	// TODO(nmittler): Add support for shared control plane.
+)
```

Comment:
These topologies are fine for now. We may need to revisit in the future. A single mesh could use shared (within region) plus replicated (across regions).